scripts/tls_test.c

```diff
@@ -38,7 +38,27 @@ static void usage(FILE *out, char *cmd)
 {
 	fprintf(out, "usage:\n");
 	fprintf(out, "  %s --connect <address> --port <port> [--key <key] [--cert <file>]+ [--times <n>]\n", cmd);
-	fprintf(out, "  %s --listen <address> --port <port> --key <key> [--cert <file>]+ [--times <n>]\n", cmd);
+	fprintf(out, "  %s --listen <address> --port <port> --key <key> [--cert <file>]+ [--cacert <file>]+ [--times <n>]\n", cmd);
+
+	fprintf(out, "\n\nOptions:\n");
```

Comment:
`pki` and `swanctl` both don't add a separator between usage lines and the option descriptions (which are indented more, though - but strangely not the same, `pki` indents with 8 spaces, `swanctl` with 11, not sure why, or why both indent the options that much at all, even without separator, less should be enough). I'm not opposed to adding a separator and, in particular, indenting the options less. But maybe make `options:` lower case (to match `usage:`, which is lower case in the other tools too) with only a single empty line before, or use a single empty line as separator and indent the options by 4 spaces (and reduce the space between options and their descriptions a bit).